modules/microshift-low-latency-tuned-profile.adoc

```diff
@@ -0,0 +1,101 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-low-latency.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-low-latency-tuned-profile_{context}"]
+= Configuring the {microshift-short} TuneD profile
+
+Configure a TuneD profile for your host to use low latency with {microshift-short} workloads using the `microshift-baseline-variables.conf` configuration file provided in the {op-system-base-full} `/etc/tuned/` host directory after you install the `microshift-low-latency` RPM package.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* You installed the `microshift-low-latency` RPM package.
+* Your {op-system} host has TuneD installed. See link:https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/monitoring_and_managing_system_status_and_performance/getting-started-with-tuned_monitoring-and-managing-system-status-and-performance#the-location-of-tuned-profiles_getting-started-with-tuned[Getting started with TuneD] (RHEL documentation).
+
+[TIP]
+====
+Using a predefined TuneD profile created for your use case can save you the time of conducting a detailed system analysis.
+====
+
+.Procedure
+
+. You can use the default `microshift-baseline-variables.conf` TuneD profile in the `/etc/tuned/` directory profile, or create your own to add more tunings.
++
+.Example `microshift-baseline-variables.conf` TuneD profile
+[source,text]
+----
+# Reserve 1 core per socket for housekeeping, isolate the rest
+isolated_cores=2-7 # <1>
+
+# Size of the hugepages
+hugepages_size=2M # <2>
+
+# Number of hugepages
+hugepages=0
+
+# Additional kernel arguments
+additional_args= # <3>
+
+# CPU set to be offlined
+offline_cpu_set= # <4>
+----
+<1> The `isolated_cores=` variable controls which cores should be isolated. By default, 1 core per socket for housekeeping is reserved in {microshift-short}. The other cores are isolated. Valid values are a core list or range. You can isolate any range, for example: `isolated_cores=2,4-7` or `isolated_cores=2-23`.
++
+[IMPORTANT]
+====
+You must keep only one `isolated_cores=` variable.
+====
++
+[NOTE]
+====
+The Kubernetes CPU manager can use any CPU to run the workload except the reserved CPUs defined in the kubelet configuration. For this reason it is best that:
+
+* The sum of the kubelet's reserved CPUs and isolated cores include all online CPUs.
+* Isolated cores are complementary to the reserved CPUs defined in the kubelet configuration.
+====
+<2> Size of the hugepages. Valid values are 2M or 1G.
+<3> Additional kernel arguments, for example, `additional_args=console=tty0 console=ttyS0,115200`.
+<4> The CPU set to be offlined.
```

Comment:
In the render numbering is broken - they appear as 1, 2, 3 and stop matching the lines in the source